refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest-client-starter/pom.xml

```diff
@@ -3,20 +3,21 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>refarch-s3-integration-client-starter</artifactId>
-    <name>refarch-s3-integration-client-starter</name>
 
     <parent>
         <groupId>de.muenchen.refarch</groupId>
         <artifactId>refarch-s3-integration</artifactId>
         <version>1.1.0-SNAPSHOT</version>
     </parent>
```

Comment:
Maybe a dedicated parent module `refarch-s3-integration-rest` would be suited better here? Applies to all other submodules as well. Won't comment any further.

Reply:
Done